tests/tests/test-rpc/test-starknet-rpc.ts

```diff
@@ -541,4 +541,39 @@ describeDevMadara("Starknet RPC", (context) => {
       expect(error.message).to.equal("40: Contract error");
     }
   });
+
+  it("pendingTransactions", async function () {
+    await rpcTransfer(
+      providerRPC,
+      ARGENT_CONTRACT_NONCE,
+      ARGENT_CONTRACT_ADDRESS,
+      MINT_AMOUNT
+    );
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const txs: { type?: any; sender_address?: string; calldata?: string[] }[] =
+      await providerRPC.getPendingTransactions();
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const tx: { type?: any; sender_address?: string; calldata?: string[] } =
+      txs[0];
+
+    expect(txs.length).equals(1);
+
+    expect(tx.type).to.be.equal("INVOKE");
+    expect(tx.sender_address).to.be.equal(toHex(ARGENT_CONTRACT_ADDRESS));
+    expect(tx.calldata).to.deep.equal(
+      [
+        1,
+        FEE_TOKEN_ADDRESS,
+        hash.getSelectorFromName("transfer"),
+        0,
+        3,
+        3,
+        ARGENT_CONTRACT_ADDRESS,
+        MINT_AMOUNT,
+        0,
+      ].map(toHex)
+    );
+  });
```

Comment:
I'd rather have an `expected_results` or `expected_pending_txs` object and assert only only the full result against the full expected result. Otherwise we're only partially testing the output